services/app-web/src/pages/StateSubmission/RateDetails/V2/SingleRateFormFields.tsx

```diff
@@ -536,17 +536,17 @@ export const SingleRateFormFields = ({
                                         </div>
                                     )
                                 )}
-                            <Button
+                            <button
```

Comment:
There is a react-uswds outline button too - not sure if that would work here but that would be first try to keep using the design system https://trussworks.github.io/react-uswds/?path=/story/components-button--outline

Reply:
Good catch!